### PR TITLE
Feature/filestorage sharetype showaccess

### PR DESCRIPTION
**Pull request by ehdou**
Add support for Share Type Show Access
For #140

### CODE REVIEW

**Comment by ehdou** [comment]
[Show access function]( https://github.com/openstack/manila/blob/16c522bf5e5c3d37018caef11dd7852462193327/manila/api/v2/share_types.py#L215) for share types

**Comment by ehdou** [comment]
@jrperritt This PR is ready for review. It's based on [PR#148](https://github.com/gophercloud/gophercloud/pull/148) and adds support for share type Show Access

**Comment by ehdou** [comment]
@jrperritt this PR is ready for review

File: openstack/sharedfilesystems/v2/sharetypes/testing/requests_test.go [hunk 1]
```diff
@@ -161,3 +161,14 @@ func TestUnsetExtraSpecs(t *testing.T) {
...
 }
+
+// Verifies that it is possible to see the access for a share type
+func TestShowAccess(t *testing.T) {
+	th.SetupHTTP()
+	defer th.TeardownHTTP()
+
+	MockShowAccessResponse(t)
+
+	_, err := sharetypes.ShowAccess(client.ServiceClient(), "shareTypeID").Extract()
```
Comment (jrperritt) [inline openstack/sharedfilesystems/v2/sharetypes/testing/requests_test.go]: Checking the error is not enough: we want to make sure that the actual value equals the expected value.

**Comment by ehdou** [comment]
@jrperritt The test is fixed, thanks for your comment

**Pull request merged**